manifests/config.pp

```diff
@@ -127,6 +127,27 @@
 
   case $::osfamily {
     'Debian': {
+      if versioncmp($::operatingsystemmajrelease, '16.04') >= 0 {
```

Comment:
Looks like this variable is not in many/any of the debian tests causing unit test failures.